docs/contributor/voiceover.rst

```diff
@@ -0,0 +1,166 @@
+
+Voiceovers help make Oppia's lessons more accessible to non-native speakers. In addition, they give a touch of personality and introduce another dimension to a lesson.
+
+In this section, you will learn how to:
+
+ * :ref:`Apply to do a voiceover <voiceoverapplication>`
+ * :ref:`Record voiceovers for a lesson <recordvoiceover>`
+
+Don't forget to check out :ref:`these guidelines <guidelines>` for recording a good voiceover.
+
+Watch the video to learn how to contribute voiceovers, or if you prefer reading the instructions instead, keep scrolling down.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/0LabiUz9hp4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+.. _voiceoverapplication:
+
+Applying to do a voiceover
+----------------------------
+
+If you want to contribute voiceovers to lessons, you will first need to apply to do so by submitting a recording from an exploration card for review. If your recording is approved and accepted, you will be able to record voiceovers in Oppia. 
+
+1. Click on the menu header **Voiceover** in the Community Dashboard.
+
+.. figure:: /images/voiceover_screen.png
+   :alt: Voiceover screen
+   :scale: 30 %
+
+   *Fig. 1*
+
+2. Select the language from the drop-down menu. You will see the chapters available for contributing voiceovers in that language.
+3. To submit a voiceover application, click the |requestvoiceover| button.
+
+.. |requestvoiceover| image:: /images/request_voiceover.png
+                      :alt: Request Voiceover button
+                      :scale: 30 %
+
+4. You will see the first card of the exploration. Use any audio recording software of your choice to record a voiceover of this card's content.
+
+.. figure:: /images/introductory_card_voiceover.png
+   :alt: Submit voiceover application
+   :scale: 35 %
+
+   *Fig. 2*
+
+5. To upload the audio file, click **Choose File**, select your file and click |submit|.
+
+.. |submit| image:: /images/submit_button.png
+            :alt: Submit button
+            :scale: 45 %
+
+.. note::
+
+   Currently Oppia only supports the MP3 file format for the audio file.
+
+6. Your application has now been sent for review. You will not be able to submit any more voiceovers for this or any other lesson until your review is accepted.
+
+7. To check the status of the review, go to the **My Contribution** page, and click on Voiceover Applications. Once your application has been accepted, you can proceed with recording a voiceover for the remaining cards in the exploration.
+
+.. figure:: /images/voiceover_accepted.png
+   :alt: Voiceover accepted status
+
+   *Fig. 3*
+
+.. _recordvoiceover:
+
+Recording voiceovers
+----------------------
+
+1. To begin recording a voiceover, click the **Voiceover** button next to your accepted application card in the **My Contribution** page. You will be taken to the **Translation** screen.
```

Comment:
This is going to be changed as well as, users will have to claim a task before doing this. There is going to be a "Claim a task" step between "Applying to do a voiceover" & "Recording voiceovers".

Reply:
I added this in, will change screenshots once this is available. I also assumed that the '**Translation**' screen in the lesson editor page will be '**Voiceover**', so I made that change as well. Let me know if not correct...